Kunena/Kunena-Forum
# Add option to use Joomla "Receive System Emails"

Pull Request for Issue #6006
 
#### Summary of Changes 

* New admin config option "Treat as System Email" for subscriptions
* Language strings for en-GB and pl-PL, placeholders for other language packs

#### Testing Instructions

* Update existing kunena/install fresh, subscribe to a thread/category
* Use another account to post in the above thread/category
  * By default or when "Treat as System Email" is set to "No" notifications should be sent to all subscribers
  * When "Treat as System Email" is set to "Yes" notifications will only be sent to subscribers whose Joomla account is configured to "Receive System Emails"
